COMPILING

```diff
@@ -122,6 +122,9 @@ Follow these instructions:
 
    cd cbmc-git/src
    make minisat2-download
+  make libzip-download
```

Comment:
Would you mind adjusting the indentation (I believe you might have used tabs) to match the surrounding text?